packages/host/src/Host.tsx

```diff
@@ -42,6 +44,11 @@ const queryClient = new QueryClient({
   },
 });
 
+Bugsnag.start({
+  apiKey: 'a09ce9e95c27ac1b70ecf3c311e684ab',
```

Comment:
thought to encode in here, rather than put into a config file - it's required to be in client side code, and being in here is at least more obfuscated than in a `config.js` file. There's little risk in this going out anyway - and is unlikely to change.